src/clr-ui/variables/_variables.scss

```diff
@@ -0,0 +1,3 @@
+// TODO: these files need to be moved out of clr-angular and into a clr-ui directory
+@import './../../clr-angular/color/variables.color';
```

Comment:
I like that this migration has started. Ultimately, wouldn't we want the actual files to live in `clr-ui` and for `clr-angular` and others to import them as is being done here?

What's our strategy there?

Reply:
Yeah thats the plan, this was just so the project was set up and building for the clr-dev app. I didn't want to move anything until the CSS custom property branch was merged.